prime-dotnet-webapi/Controllers/AuthorizedUsersController.cs

```diff
@@ -0,0 +1,198 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Auth;
+using Prime.Extensions;
+using Prime.Models;
+using Prime.Models.Api;
+using Prime.Services;
+using Prime.ViewModels;
+using Prime.ViewModels.Parties;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/parties/authorized-users")]
+    [ApiController]
+    [Authorize(Roles = Roles.PrimeEnrollee + "," + Roles.ViewSite)]
+    public class AuthorizedUsersController : ControllerBase
```

Comment:
None of the endpoints are checking that the user has permission to access this Party (by UserId)

Reply:
@james-hollinger guessing your looking for `authorizedUser.Party.PermissionsRecord().AccessableBy(User)`, and looking through it a bit more guess I could also be using `!authorizedUser.Validate(User)`.  Have one question then will update this tomorrow.